app/policies/nav_bar_policy.rb

```diff
@@ -3,9 +3,15 @@ def visible_buttons
     visible = Set.new
     visible << (user ? 'Logout' : 'Login')
     visible << 'Sign Up' unless user
-    visible << 'Feedback' if user
+    visible << 'Feedback' if is_admin?(user)
     visible.merge %w[Glossary Contributions] if system_settings.peer_to_peer?
     visible.merge %w[Contributions Matches Admin] if can_admin?
     visible.to_a
   end
+
+  private
+
+  def is_admin?(user)
+    user && (user.admin_role? || user.sys_admin_role?)
+  end
```

Comment:
The `ApplicationPolicy` base class provides a `can_admin?` method that serves the same purpose. In fact, looking at line 8 above, you can probably add `Feedback` to the array there instead.

Reply:
Indeed—thanks for pointing that out (plus a little 🤦 😄)
Updated/simplified (e139258)

Anything else?

Thanks!